meta-spec-0000.md

```diff
@@ -26,9 +26,8 @@ Projects decide for themselves whether to adopt any given SPEC—often, this
 would be through team consensus.
 A SPEC may not be a good fit for every single project, and thus there is no
 expectation that all SPECs must be adopted by all projects.
-Different SPECs may even recommend differing approaches to the same problem.
-In this case, some projects may adopt one of the competing SPECS,
-while some projects adopt the other one.
+Different SPECs may even recommend differing approaches to the same problem for
+the sake of discussion and consensus-building among the projects.
```

Comment:
> In this case, some projects may adopt one of the competing SPECS, while some projects adopt the other one.

My sense was that this might be too specialized a situation to bring up here without an example. At face value it's difficult to see how this meshes with unifying across the ecosystem. You could add an example, but that might not fit very well with the high-level nature of the rest of the document.

Reply:
I've been trying to figure out how to say what I want to say and would be happy for suggestions.

It may already be implied, but the thing I want to emphasize is that this is not an organization that can enforce (or even should enforce) rules.  I want everyone to do the same thing, but I also don't want this to devolve into a flame war.  I suspect there will be many occasions where there are strong diverging opinions.  In such instances, I am not sure that the SPECs should arbitrarily choose one opinion.  And I think there is value in documenting multiple opinions.  I am not sure how that will always work.  I also don't think it should be the job of the SSC to choose between competing SPECs.  The unification will have to be done organically by individual projects agreeing to adopt the same practices.

If we only allow SPECs that everyone can agree on then I am not sure how much benefit the SPECs will serve.  Documenting two recommended approaches is still better than having each project invent its own approach.  It may even be desirable to have multiple solutions to shared problems.

That said, I am not sure it needs to be said.  Maybe we don't need to mention it now and we can wait for the situation to arise.  Or maybe just saying
```
Different SPECs may even recommend differing approaches to the same problem for
the sake of discussion and consensus-building among the projects.
```
is sufficient without adding projects can adopt the one the want (since that is already true).

Reply:
I am just going to remove it all.  We should wait until the situation arises and then see what makes sense.  Once we have experience to guide us, we can add more detail (if needed).

@rosbarr I will create a new SPEC category called `Questions`.  And I will try to briefly summarize the issue there as a new topic.